os/developer-guides.md

```diff
@@ -0,0 +1,18 @@
+# Developer Guides
+
+Most users will never have to build Container Linux from source or modify it in any way. If you have a need to modify Container Linux, we provide an SDK that allows you to build your own developer images. We also provide OEM functionality for cloud providers and other companies that must customize Container Linux to run within their environment.
+
+* [Modifying Container Linux][mod-cl]
```

Comment:
Maybe we should toss in a link to mantle, or the kola subsection of it (https://github.com/coreos/mantle/#kola).

The 'modifying CL' page links to cork, so the other big mantle piece that could be useful is kola, which would be `[Testing Container Linux](https://github.com/coreos/mantle/#kola)`.